build/followers/render.php

```diff
@@ -64,8 +64,10 @@ function ( $follower ) {
 		return array(
 			'handle' => '@' . $username,
 			'icon'   => $actor->get_icon(),
-			'name'   => $actor->get_name() ?? $username,
-			'url'    => object_to_uri( $actor->get_url() ) ?? $actor->get_id(),
+			// phpcs:ignore Universal.Operators.DisallowShortTernary.Found
```

Comment:
Let's add an exception for this in `phpcs.xml` and remove any remaining ignores. [The reasons for Core to not allow it](https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/) don't apply to this plugin.